style.css

```diff
@@ -14,6 +14,14 @@ body {
   width: 375px;
 }
 
+article {
+  margin-bottom: 3.5em;
+  margin-top: 2em;
+  padding-bottom: 3em;
+  padding-top: 3em;
+  width: 375px;
+}
+
 header {
   display: flex;
   justify-content: space-between;
```

Comment:
- [OPTIONAL ] In my opinion It would be better if you used classes instead of calling tags, since this a long project, It could avoid you to get lost in youir codes